region/brinstar/red.json

```diff
@@ -1624,6 +1624,22 @@
                   "name": "Base",
                   "notable": false,
                   "requires": ["h_canBombThings"]
+                },
+                {
+                  "name": "Jump and Speedball",
+                  "notable": false,
+                  "requires": [
+                    "Spazer",
+                    "canMockball",
+                    "canCarefulJump",
+                    {"canComeInCharged": {
+                      "framesRemaining": 180,
+                      "shinesparkFrames": 0,
+                      "fromNode": 3,
+                      "unusedTiles": 1
+                    }}
+                  ],
+                  "note": "Shoot forward with spazer to break the shot block, then jump into the morph tunnel while staying blue to break a bomb block."
```

Comment:
does wave+plasma work?

Reply:
It does not

Reply:
Wave+Plasma does work but it's not reliable.
I could add a notable for jump then shoot to break the block with any beam.

Reply:
Either way. It could be added, a note, or a fixme, depending on how challenging it is and if you want it as part of this PR